CMakeLists.txt

```diff
@@ -272,6 +272,7 @@ add_library(PKWare STATIC
 set(devilutionx_SRCS
   Source/appfat.cpp
   Source/automap.cpp
+  Source/automap_render.cpp

```

Comment:
Should this live in `engine/` with the game world render?

Reply:
I'm not sure what `engine.h` is currently -- it seems to do quite a few things besides rendering.

Can I leave the location up to you for a follow-up refactoring? :)

Reply:
not engine.h, engine/ (the folder)

Reply:
also it dosnt seam to be strictly tied to thee auto map, so how about `Source/engine/render/lines.cpp`

Reply:
It is strictly tied to the automap; Why have `engine/render` and not just `render/`? Do we render anything outside of the "engine" (whatever the "engine" means)?

Reply:
> The core functionality typically provided by a game engine may include a rendering engine ("renderer") for 2D or 3D graphics, a physics engine or collision detection (and collision response), sound, scripting, animation, artificial intelligence, networking, streaming, memory management, threading, localization support, scene graph, and video support for cinematics.

Basically anything that doesn't distinguish the game itself.

We could have all of this in the base Source folder, but it's a way for us to segregate the "complicated stuff" that some one just looking to modify the game-play probably has no need to look at, and which is allowed to be a bit more complicated then the rest of the code-base.